src/app/zap-templates/zcl/data-model/chip/thermostat-cluster.xml

```diff
@@ -0,0 +1,137 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2021 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="HVAC"/>
+  <cluster>
+    <name>Thermostat</name>
+	<domain>HVAC</domain>
+    <description>An interface for configuring and controlling the functionality of a thermostat.</description>
+    <code>0x0201</code>
+    <define>THERMOSTAT_CLUSTER</define>
+    <client tick="false" init="false">true</client>
+    <server tick="false" init="false">true</server>
+    <globalAttribute side="either" code="0xFFFD" value="3"/>
+    <attribute side="server" code="0x0000" define="LOCAL_TEMPERATURE" type="INT16S" min="0x954D" max="0x7FFF" writable="false" reportable="true" optional="false">local temperature</attribute>
```

Comment:
If we are going to create a separate xml file for thermostat server we should have values in decimal as that has been adopted to be the matter standard.

Reply:
All xml files use hex right now.